python/hail/dataset.py

```diff
@@ -2745,6 +2745,52 @@ def linreg_multi_pheno(self, ys, covariates=[], root='va.linreg', use_dosages=Fa
                                            jarray(Env.jvm().java.lang.String, covariates), root, use_dosages, min_ac,
                                            min_af)
         return VariantDataset(self.hc, jvds)
+    
+    @handle_py4j
+    @requireTGenotype
+    @typecheck_method(ys=listof(strlike),
+                      covariates=listof(strlike),
+                      root=strlike,
+                      use_dosages=bool,
+                      variant_block_size=integral)
+    def linreg3(self, ys, covariates=[], root='va.linreg', use_dosages=False, variant_block_size=16):
```

Comment:
Just to clarify, I don't expect anyone to use this outside of UKBB or others who come with extreme use cases, we have no obligation expose this stuff in 0.1, the sooner we move all development to 0.2 the better, and when that happens, I plan to rip out linreg/linreg_multi_pheno, rename linreg3 => linreg and make it generic (not genotype-specific) in which case the ac/af interface doesn't make sense and I still think filtering doesn't belong in the regression method.